priv/repo/migrations/20241211135758_certifications.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Orbit.Repo.Migrations.CertificationsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:certifications) do
+      add(:badge, :string)
+      # Enum
+      add(:type, :integer)
+      # Enum
+      add(:rail_line, :rail_line)
+      add(:expires, :date)
+      timestamps()
+    end
+
+    create unique_index(:certifications, [:badge, :type])
+    create index(:certifications, [:badge])
```

Comment:
I think that Postgres can use the prefix of an index. So if there's an index on `[:a, :b, :c]`, then it can also use it as an index on `[:a, :b]`. So this index is redundant with the `unique_index` above.

Reply:
Fixed https://github.com/mbta/orbit/pull/203/files#diff-248c4b179027894763ea37f20c43c9e2ac96ea5b22619c4e898777d809a5c8b5R15